### ISSUE TITLE
Some motions don't work, others do

### ISSUE THREAD

**Issue by eapache** [Lokaltog/vim-easymotion#171]
Started playing around, all easymotion configuration at its default. `\\j` and `\\k` and `\\w` all work fine, but `\\s` gives me:

```
Error detected while processing function vital#of..vital#_easymotion#new..<SNR>104__import:
E605: Exception not caught: vital: module not found:
```

Google didn't turn up anything obvious. Ideas?

vim 7.4.373 on Ubuntu x64

**Comment by eapache** [comment]
I just came back to this (renamed some things and accidentally re-introduced it) and I am having trouble understanding why the `tolower` call (around line 161 of `_easymotion.vim`) is there in the first place... for case-insensitive paths it doesn't matter, and for case-sensitive paths we should already have the path in the right case.

Simple removing the `tolower` call fixes this issue properly, and unless it causes some other bug I would suggest removing it entirely.

**Comment by haya14busa** [comment]
Sorry, https://github.com/Lokaltog/vim-easymotion/blob/607fb0e6eea76e829cf9c2119e6ff907bf2219b0/autoload/vital/_easymotion.vim#L164 is the code in the part of library named vital.vim, and I should not modify the library code for the sake of maintainability.

I guess `tolower()` is for the case-insensitive paths system like Windows, but I also doubt that `tolower()` is necessarily. I'll ask the library maintainer, thanks.

**Comment by haya14busa** [comment]
Could you please tell me your runtimepath directory name and the file system?

**Comment by haya14busa** [comment]
> Could you please tell me your runtimepath directory name and the file system?

Sorry, I understood the issue.

**Comment by haya14busa** [comment]
Could you please trying this branch? https://github.com/Lokaltog/vim-easymotion/compare/fix-vital-case

It should fix the problem but I cannot confirm it.

**Comment by eapache** [comment]
Yes, that fixes the problem, thank you!

**Comment by haya14busa** [comment]
I merged it into master now. Thank you @eapache! :sparkles:  https://github.com/Lokaltog/vim-easymotion/commit/8acdfc60e58bb0600ded42a4f752bec6e3b6d882

**Comment by myriasofo** [comment]
Hi there, I'm getting a very similar error (I can make a separate issue if that's helpful). As with the user eapache, `\\j` and others work fine, while the family of `\\f` don't work at all.
 
    Error detected while processing function <SNR>55_import..<SNR>55__import:
    line    6:
    E605: Exception not caught: vital: module not found: Over.Commandline.Base

Here are some notes:
* My error says `55_import`, while for eapache it was `97_import`. Also note that module not found is Over.Commandline.**Base**, as opposed to just Over.Commandline.
* I don't really understand what eapache meant when talking about symlinks. My setup seems normal: I have my vimrc in my User folder (I'm on Windows 7). I use vundle, but I also tried manually putting in easymotion's files to their respective folders (ie. copied all of the `autoload` folder into vim's main autoload folder, etc.)
* Strangely, `\\f` is working on my laptop - it's just not functioning on my work desktop. (Both are running Windows 7)

Some additional points
* I tried using a clean vimrc and that didn't help.
* I tried turning off tons of easymotion options and that didn't help.
* My easymotion version is from 4/27/15. I will try updating it when I get home.
